pkg/cmd/server/api/helpers.go

```diff
@@ -26,6 +27,18 @@ func init() {
 	for _, feature := range KnownOpenShiftFeatures {
 		knownOpenShiftFeatureSet[strings.ToLower(feature)] = feature
 	}
+	for alias, feature := range FeatureAliases {
+		knownOpenShiftFeatureSet[strings.ToLower(alias)] = feature
+	}
+}
+
+// NormalizeOpenShiftFeature returns canonical name for given OpenShift feature
+// flag or an alias. Returns empty string if nothing matches.
```

Comment:
why return empty string for unknown features? usage would be a lot simpler with this impl:
```
name = strings.ToLower(name)
if feature, ok := knownOpenShiftFeatureSet[name]; ok {
  return feature, true
}
return name, false
```